### DISCUSSION TITLE
Methods of adding Libraries like firebase-bom.

### DISCUSSION THREAD

**Discussion by H-uey** [Sketchware-Pro/Sketchware-Pro#1268] [Ideas]
**Is your feature request related to a problem? Please describe.**
To ease updating Firebase libraries, using Firebase-bom would be perfect. Manually adding and excluding firebase libs is causing inconsistencies and app crashes due to conflict. This can also solve issues to do with analytics and crashlytics.
**Describe the solution you'd like**
An implementation to handle this is that the library downloader is not importing this so natively supporting it would be nice.
**Describe alternatives you've considered**
I'm stuck.
**Additional context**
Just firebase. Maybe a feature to update it just like the existing way of adding Firebase by toggling on at the library.

**Comment by hasrat-ali** [comment]
Since we lack contributors, we can't do this for now.

**Reply by CarlosCMD** [reply]
> Since we lack contributors, we can't do this for now.

Any workarounds or updated firebase libraries with a recent version and doesn't crash due to newer dependencies?

**Reply by j-px** [reply]
Follow this: https://docs.sketchware.pro/builtinlib

**Comment by PranavPurwar** [comment]
BOMs or Bill of Materials are not considered "libraries" per se, rather they just specify versions of other dependencies. There is no point of being able to "add" them as such.
